### DISCUSSION TITLE
Add uboss

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#555] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
since pull requests are confusing:

 Add
 
 
 This exploit only works for iboss, cisco umbrella, blocksi and securly (maybe). In order to do this, you first must go to https://tinyurl.com/bhcuboss and copy the code from there, and put javascript: infront of it and then paste it into a bookmark. Then, you can go to:
https://tinyurl.com/byeswamp for IBoss
https://tinyurl.com/blockboss for Blocksi
https://tinyurl.com/cyouboss for Cisco Umbrella (may not work)
https://tinyurl.com/suboss for securly (may not work)
Click on the blue link in the page.
Then, click the bookmark you made earlier!
Join the Blue Hat Crew (dsc.gg/blue-hat-crew or https://discord.gg/GdzCn3hT6N) or dm Cubing_Hay#3705 if it doesn't work!

**Comment by Aka-but-nice** [comment]
> since pull requests are confusing:
> 
> Add
> 
> This exploit only works for iboss, cisco umbrella, blocksi and securly (maybe). In order to do this, you first must go to https://tinyurl.com/bhcuboss and copy the code from there, and put javascript: infront of it and then paste it into a bookmark. Then, you can go to: https://tinyurl.com/byeswamp for IBoss https://tinyurl.com/blockboss for Blocksi https://tinyurl.com/cyouboss for Cisco Umbrella (may not work) https://tinyurl.com/suboss for securly (may not work) Click on the blue link in the page. Then, click the bookmark you made earlier! Join the Blue Hat Crew (dsc.gg/blue-hat-crew or https://discord.gg/GdzCn3hT6N) or dm Cubing_Hay#3705 if it doesn't work!

Bro I just said it wouldn't work for Securly, Cisco umbrella, it only works for Blocksi, and iBoss.
